tpm/structures.go

```diff
@@ -67,6 +72,18 @@ type pcrInfo struct {
 	DigestAtCreation digest
 }
 
+type nvDataPublic struct {
+	tag          tpmutil.Tag
+	index        tpmutil.Handle
+	pcrshort1    pcrInfoShort
```

Comment:
Go style requires this and the next name to be "pcrShort1". However, I don't want to encode type information in field names, either. Please rename this and the next field to remove the type information ("short").